### PR TITLE
chimeric_contig / summary_report

### PR DESCRIPTION
**Pull request by alyssa-ab**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/viralintegration/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/viralintegration _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: modules/local/chimeric_contig_evidence_analyzer.nf [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    container "trinityctat/ctat_vif"
+
+    input:
+    tuple val(meta), path(bam), path(bai), path(gtf)
+
+    output:
+    tuple val(meta), path ("*.evidence_counts.tsv") , emit: evidence_counts
+    tuple val(meta), path ("*.evidence.bam")        , emit: evidence_bam
+    tuple val(meta), path ("*.evidence.bam.bai")    , emit: evidence_bai
+    path ("insertions_validated.txt") , emit: txt
```
Comment (edmundmiller) [inline modules/local/chimeric_contig_evidence_analyzer.nf]: Does this one need a meta, and line up the `,`

File: workflows/viralintegration.nf [hunk 2]
```diff
@@ -63,8 +65,10 @@ include { STAR_GENOMEGENERATE as STAR_GENOMEGENERATE_HOST
           STAR_GENOMEGENERATE as STAR_GENOMEGENERATE_PLUS } from '../modules/nf-core/star/genomegenerate/main'
 include { STAR_ALIGN as STAR_ALIGN_HOST
           STAR_ALIGN as STAR_ALIGN_PLUS } from '../modules/nf-core/star/align/main'
-include { SAMTOOLS_SORT               } from '../modules/nf-core/samtools/sort/main'
-include { SAMTOOLS_INDEX              } from '../modules/nf-core/samtools/index/main'
+include { SAMTOOLS_SORT
+          SAMTOOLS_SORT as SAMTOOLS_SORT_VALIDATE} from '../modules/nf-core/samtools/sort/main'
```
Comment (edmundmiller) [inline workflows/viralintegration.nf]:
```suggestion
          SAMTOOLS_SORT as SAMTOOLS_SORT_VALIDATE } from '../modules/nf-core/samtools/sort/main'
```

File: tests/modules/summary_report/main.nf.test [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    test("Should run without failures") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test', single_end:false ],
+                    file('https://raw.githubusercontent.com/nf-core/test-datasets/viralintegration/testdata/summary_report/test.vif.evidence.bam', checkIfExists: true),
+                    file('https://raw.githubusercontent.com/nf-core/test-datasets/viralintegration/testdata/summary_report/test.vif.evidence.bam.bai', checkIfExists: true),
+                    file('https://raw.githubusercontent.com/nf-core/test-datasets/viralintegration/testdata/summary_report/test.vif.init.filtered.abridged.tsv', checkIfExists: true),
```
Comment (edmundmiller) [inline tests/modules/summary_report/main.nf.test:16]: These need to updated to the fixed results

**Review by edmundmiller — APPROVED** [review]
LGTM! Only thing is you might want to update the summary_report test data or make a follow up issue.

File: tests/modules/summary_report/main.nf.test (hunk 3 above)
Comment (alyssa-ab) [reply tests/modules/summary_report/main.nf.test:16]: https://github.com/nf-core/viralintegration/issues/37

**Pull request merged**